app/models/user.rb

```diff
@@ -105,6 +105,8 @@ def eligible_sysadmin?
     return true if superuser || sysadmin
   end
 
+  def tester_trainer; end
```

Comment:
Was this an intended method?

Reply:
Thanks @carolyncole for the suggested change